.github/workflows/ci.yml

```diff
@@ -70,6 +70,57 @@ jobs:
           path: /github/home/go/src/github.com/BitBoxSwiss/bitbox-wallet-app/frontends/android/BitBoxApp/app/build/outputs/apk/debug/app-debug.apk
           name: BitBoxApp-android-${{github.sha}}.apk
           if-no-files-found: error
+
+  e2e-android:
+    name: Android E2E Tests
+    needs: android
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        api-level: [30, 31, 33, 34, 35]
```

Comment:
32 missing?

Reply:
I was thinking of only targeting API that were introduced with new Android versions, so I skipped 32 (which is part of Android 12 like level 31) but I changed it now